pcsx2/ps2/BiosTools.cpp

```diff
@@ -365,7 +370,10 @@ void LoadBIOS()
 		}
 
 		if (CurrentBiosInformation == NULL)
-			Console.Warning("BIOS Warning: Unknown BIOS version. The debugger thread and stack frame views will not be functional.");
+			Console.Warning("BIOS Warning: Unknown BIOS version. The debugger thread and stack frame views will not be functional.\n(This is harmless for 90\% of users)");
```

Comment:
Do we really need to add this? Lambda users won't read our emuLog probably

Reply:
I'm unsure what lambda you're referring too but if I can make it silently fail I guess.